src/app/shared/font-picker/font-picker.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'mifosx-font-picker',
+  templateUrl: './font-picker.component.html',
+  styleUrls: ['./font-picker.component.scss']
+})
+export class FontPickerComponent implements OnInit {
+
+  currentFont = 'roboto';
+
+  fonts = [
+    'roboto',
+    'monospace',
+    'anton'
+  ];
+
+  constructor() { }
+  ngOnInit() {
+    this.currentFont = localStorage.getItem('mifosXfont');
+    document.body.classList.add(this.currentFont);
+  }
+
+
```

Comment:
Subscribe to Value Changes of Form Control and change the font of the app accordingly.